tst/CTA.Rules.Test/Models/WebFormsFullExpectedOutputConstants.cs

```diff
@@ -353,10 +357,10 @@ protected override void OnInitialized()
             Model = CatalogService.GetCatalogItemsPaginated(DefaultPageSize, DefaultPageIndex);
             PeopleModel = new List<People>()
             {new People(""Andy"", ""Wayne"", ""PG""), new People(""Bill"", ""Johnson"", ""SD""), new People(""Caroline"", ""Barry"", ""Manager"")};
-            PeopleGrid.DataSource = PeopleModel;
-            PeopleGrid.DataBind();
-            productList.DataSource = Model.Data;
-            productList.DataBind();
+            PeopleGrid_DataSource = PeopleModel;
+             // PeopleGrid.DataBind();
```

Comment:
why are these commented

Reply:
Methods are not supported for conversion, and while the DataBind() call is actually valid on BlazorWebFormsComponents components (like grid view) the method call doesn't do anything in Blazor. Basically invalid member accesses just get commented out and we convert all the rest